src/e2e-test/resources/pluginParameters.properties

```diff
@@ -135,3 +135,42 @@ gcsMoveReadRecursivePath=testdata
 gcsMoveReadRecursiveSubDirectory=testdata/GCS_RECURSIVE_TEST
 locationEU=eu
 ## GCSMove-PLUGIN-PROPERTIES-END
+
+## DATAPLEX-PLUGIN-PROPERTIES-START
+dataplex.SourceSchema=[{"key":"transaction_info","value":"boolean"},{"key":"transaction_num","value":"bytes"},\
+  {"key":"transaction_uid","value":"string"},{"key":"transaction_time","value":"time"},{"key":"latitude","value":"double"},\
+  {"key":"unique_key","value":"long"},{"key":"business_ratio","value":"decimal"},{"key":"inputs","value":"record"},\
+  {"key":"business_bigratio","value":"string"},{"key":"committer","value":"record"},{"key":"Userdata","value":"record"}]
+dataplex.orcEntity_Schema=[{"key":"transaction_info","value":"boolean"},{"key":"transaction_num","value":"bytes"},\
+  {"key":"transaction_uid","value":"string"},{"key":"transaction_date","value":"date"},{"key":"transaction_time","value":"time"},\
+  {"key":"latitude","value":"double"},{"key":"unique_key","value":"long"},{"key":"business_ratio","value":"decimal"},\
+  {"key":"updated_on","value":"timestamp"},{"key":"business_bigratio","value":"string"}]
+dataplex.project=cdf-athena
```

Comment:
use camel case instead of `.` in all the properties.

Reply:
All properties updated with camel case.